src/angular-google-analytics.js

```diff
@@ -187,13 +192,28 @@
         };
 
         var _ga = function() {
+
           if (!$window.ga) {
             that._log('warn', 'ga function not set on window');
           }
           if (logAllCalls === true) {
             that._log.apply(that, arguments);
           }
-          $window.ga.apply(null, arguments);
+          // If multiple trackers && arguments valid && command supports multiple trackers
+          // Temporary check if command contains '.' until all functions are switched
+          if (angular.isArray(accountId) && arguments && arguments.length > 1 && arguments[0].indexOf('.') < 0 &&
+              trackingSupportingMultipleTrackers.indexOf(arguments[0]) >= 0) {
+            var args = Array.prototype.slice.call(arguments);
+            // Create command for each tracker an apply
+            accountId.forEach(function (trackerObj) {
+              var command = trackerObj.name + '.' + args[0];
+              args[0] = command;
+              $window.ga.apply(null, args);
```

Comment:
This is still problematic and won't work properly. The code needs to be:

```javascript
var command = trackerObj.name + '.' + arguments[0],
      args = arguments.slice();
args[0] = command;
$window.ga.apply(null, args);
```

The justification is as follows. Setting args on line 206 to a copy of the original arguments does not allow it to be reset on each loop of the ```forEach``` on line 208. The consequence is that when ```args[0]``` is set on line 210 that new value will then be used on line 209 on each subsequent iteration of the ```forEach```. As an illustrative example:

Let, ```args = ['x', 'y', 2]``` and ```accountId = [{ name: 'name1' }, { name: 'name2' }]```

On Iteration #1 we will get:
   trackerObj = { name: 'name1' }
   command = 'name1.x'
   args = ['name1.x', 'y', 2]

On iteration #2 we will get:
   trackerObj = { name: 'name2' }
   command = 'name2.name1.x'
   args = ['name2.name1.x', 'y', 2]

I hope that makes it clear what is wrong with the implementation in this commit. Additionally, if the unit tests aren't catching this then a new unit test should be added for ensuring existing behaviors are preserved and the expected behavior of this function is maintained.